content/doc/developer/publishing/continuous-integration.adoc

```diff
@@ -8,12 +8,24 @@ It will build all plugin repositories in the `jenkinsci` organization that have
 
 The typical plugin build (Maven or Gradle) can be run by just having the following statement in the `Jenkinsfile`:
 ----
-buildPlugin()
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 17], // use 'docker' if you have containerized tests
+    [platform: 'windows', jdk: 11],
+])
 ----
 
 Gradle support in `buildPlugin()` is deprecated and will be eventually removed. Please use:
 ----
-buildPluginWithGradle()
+buildPluginWithGradle(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 17], // use 'docker' if you have containerized tests
```

Comment:
```suggestion
    [platform: 'linux', jdk: 11],
    [platform: 'linux', jdk: 17],
```

This suggestion is to be challenged: 
- The comment is misleading: requiring "Docker" would be to set `useContainerAgent` to `false`. It's a problem on the upstream archetype but I would vote to not add this to the scope here
- What is the reason for not using Linx + JDK11? (I'm not sure)

Reply:
Ref. https://github.com/jenkinsci/archetypes/pull/552

Reply:
I think the latest version of the archetype should always be regarded as the source of truth and other documentation should simply match that. Any substantive debates on what we want to recommend are better handled within the context of a pull request to the archetype.